Nodejs/Product/Nodejs/Snippets/1033/Nodejs/dowhile.snippet

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>dowhile</Title>
+      <Description>Code Snippet for a Do...While Loop</Description>
+      <Shortcut>dowhile</Shortcut>

```

Comment:
this snippet can be ```do``` rather than ```dowhile``` - it's both shorter and consistent with the rest of the VS javascript experience.